pkg/controllermanager/controller/shoot/shoot_utils.go

```diff
@@ -114,22 +114,22 @@ func BoolToStatus(cond bool) Status {
 }
 
 // ComputeStatus computes the label Status of a shoot depending on the given lastOperation, lastError and conditions.
-func ComputeStatus(lastOperation *gardencorev1alpha1.LastOperation, lastError *gardencorev1alpha1.LastError, conditions ...gardencorev1alpha1.Condition) Status {
+func ComputeStatus(lastOperation *gardencorev1alpha1.LastOperation, lastErrors []gardencorev1alpha1.LastError, lastError *gardencorev1alpha1.LastError, conditions ...gardencorev1alpha1.Condition) Status {
```

Comment:
What about `.status.lastError`?

Reply:
Hmm what do you mean? it's right after `lastErrors []gardencorev1alpha1.LastError,`